packages/components/src/navigator/use-navigator.ts

```diff
@@ -13,11 +13,12 @@ import type { Navigator } from './types';
  * Retrieves a `navigator` instance.
  */
 function useNavigator(): Navigator {
-	const [ , setPath ] = useContext( NavigatorContext );
+	const { setNavigatorPath, setIsAnimating } = useContext( NavigatorContext );
 
 	return {
 		push( path, options ) {
-			setPath( { path, ...options } );
+			setIsAnimating( true );
```

Comment:
This should be conditional to the return from `useReducedMotion` to avoid the overflow style getting applied and never removed in such contexts.

Reply:
Well spotted — Added the check in [`5a17e72` (#35317)](https://github.com/WordPress/gutenberg/pull/35317/commits/5a17e725b1a22bd936d00724f9058432488304ba)